src/server/mocked-endpoint.ts

```diff
@@ -20,4 +20,23 @@ export class MockedEndpoint implements MockedEndpointInterface {
         // Wait for all completed running requests to have all their details available
         return Promise.all<CompletedRequest>(this.rule.requests);
     }
+
+    pendingMocks(): string[] {
```

Comment:
What's the use case for this exactly? Do you have some example code showing how you're using this?

I can see a few cases, but I'm surprised that you just want a string representation of the rules that are pending. Would a structured object with the actual rule details be more useful? Alternatively, if you want a readable representation, why not the whole `explain()` value?

Reply:
I just wanted a very similar feature like 'pendingMocks' (https://www.npmjs.com/package/nock#pendingmocks) in 'nock'. It can be very useful when you have many tests with many mocks. In this case with this feature you have to do only one assertion in afterEach and you will get pending mocks so you dont need to check getSeenRequests for every mocked endpoint. Dont need much detailed description as a string with method and url (e.g. "GET /mocked-endpoint?foo=bar") because what is important that there are pending mocks and this string is enough to identify your mock. I created tests in remote-client.spec.ts with describe 'pending mocks', you can check how this feature works. In tests the assertion (e.g. "expect(pendingMocks).to.be.empty") is in test case but of course "await client.pendingMocks()" could be in afterEach too.